### DISCUSSION TITLE
[Proposition] Renommer le dépôt

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices-wordpress#18] [General] [answered]
Bonjour,
Je propose de renommer le dépôt en wordpress-best-pactrices pour une homogénéité du repos.
Bien à vous.

**Comment by dNicolle** [comment, marked as answer]
Hello @florinesueur 
Je suis d'accord pour dire que le dépot est mal nommé, mais pas par homogénéité, mais parce qu'il y a aussi un sous-repo lié aux futurs travaux
https://github.com/cnumr/fiches-wordpress-greenit/tree/main/Ecoindex%2C%20plugins%2C%20badge%2C%20CLI
Vides pour le moment, mais qui pourraient évoluer si les sujets sont pris en main.
Peut-être le renommer WordPress tout simplement, puis à l'interieur, comme c'est le cas https://github.com/cnumr/fiches-wordpress-greenit/tree/main/best-practices-WordPress

Qu'en penses-tu ?
@hrenaud et toi ?

**Reply by hrenaud** [reply]
Je n'ai pas d'affect sur le nom, juste si on le renomme, les gens vont perdre les liens :) et ils devront changer leur lien git sur leur poste 
https://linuxize.com/post/how-to-change-git-remote-url/

Donc, il faudra re-communiquer.

**Reply by dNicolle** [reply]
on va attendre la finalisation de cette v1 alors.
ok @florinesueur ?
